_providers/verizon.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Verizon
+status: OK
+domains: 
+  - verizon.net
+server:
+  - hostname: smtp.verizon.net
```

Comment:
please specify either smtp+imap or nothing.

if in doubt, just delete the provider